Documentation/project-docs/event-logging-guide.md

```diff
@@ -0,0 +1,25 @@
+#Event Logging in CoreCLR
+
+For event logging mechanism in coreclr, we are using the [ETW](https://msdn.microsoft.com/en-us/library/windows/desktop/bb968803%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)- Event Tracing for Windows as a template for all logging mechanism
+
+## How to Add Events to the Runtime:
+
+**Step1**: Edit the [manifest] (https://github.com/dotnet/coreclr/blob/master/src/vm/ClrEtwAll.man). For guidelines look at the existing events in the manifest and this guide for [ETW Manifests](https://msdn.microsoft.com/en-us/library/dd996930%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)
+
+**Step 2**:
+
+-*For Windows*: TBD
```

Comment:
It would be good to track down whoever did this the first time and get them to add instructions for Windows.